### ISSUE TITLE
Documentation or tutorial

### ISSUE THREAD

**Issue by maaamcube** [shreeve/picousb#2]
Hello @shreeve,
Thanks for this amazing job.
I want to know if you can add a tutorial or documentation to implement a vendor specific host to send vendor specific device through wifi ?
Thanks.

**Comment by shreeve** [comment]
@maaamcube - Thanks!

The library is still very new, in fact just getting USB Host working on the rp2040 with only pico-sdk was quite challenging. There is still a need to add in support for various device types, including hubs, hid, cdc, msc, etc. Hopefully, this work will be much more straightfoward since there is a good solid base, and also plenty of other code out in the wild that can be brought over.

In summary, there's nothing yet to support what you are requesting, but the fundamentals are now here and anyone who is able can add support for various USB classes.
